recipes/simplemseed/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "simplemseed" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "7c268a35441d7e85d8078ee57b9c20cf2b9fcf23481a9c3548565ffbf37cd401" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - fdsnsourceid = simplemseed.fdsnsourceid:main
+    - mseed3details = simplemseed.mseed3details:main
+    - mseed3merge = simplemseed.mseed3merge:main
+    - mseed2to3 = simplemseed.mseed2to3:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.9
+    - hatchling >=1.21.0
+    - pip
+  run:
+    - python >=3.9
+    - crc32c
+    - numpy <2.0dev0
+    - jsonpointer
+    - importlib-metadata  # [py<310]
```

Comment:
The recipe can be noarch and this is fine to install in py>=310, it is just a stub there.